careers/application-process/index.html

```diff
@@ -281,13 +281,17 @@ <h3 class="u-hide-on-mobile">Asking About your Application</h3>
     <div class="block
                 block__padded-top
                 block__border-top
+                block__flush-bottom
                 content-l">
-        <section class="content-l_col content-l_col-1-2">
+        <article class="content-l_col content-l_col-1-3">
+            {% include "_template-current-openings.html" %}
+        </article>
+        <article class="content-l_col content-l_col-1-3">
             {% include "_template-working-at-cfpb.html" %}
-        </section>
-        <section class="content-l_col content-l_col-1-2">
+        </article>
```

Comment:
I would say they aren't since standalone to me means someone would come to this page for that content. Like a blog post would be standalone or a forum post would be standalone. On this page, the information about the application process would be standalone. 

These seem like supplementary information that advertises another part of the website and very similar to information that would be included in a page footer or sidebar.